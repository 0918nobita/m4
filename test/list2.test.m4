include(list2.m4)dnl
define([list_a], [A, B, C])dnl
fst(list_a)
dnl => A
snd(list_a)
dnl => B
length(empty)
dnl => 0
length()
dnl => 1
length(5)
dnl => 1
length(5, 6)
dnl => 2
length(5, 6, empty)
dnl => 3
nth(1, [list_a])
dnl => A
nth(2, [list_a])
dnl => B
nth(3, [list_a])
dnl => C
tail()
dnl => empty
tail(10)
dnl => empty
tail(10, 11)
dnl => 11
tail(10, 11, 12)
dnl => 11,12
