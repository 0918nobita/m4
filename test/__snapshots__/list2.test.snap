A
B
0
1
1
2
3
A
B
C
empty
empty
11
11,12
